go/interpreter/lfvm/instructions_test.go

```diff
@@ -1827,6 +1827,28 @@ func TestOpBlockhash(t *testing.T) {
 	}
 }
 
+func TestOpExtCodeHash_WritesCorrectHashInStack(t *testing.T) {
```

Comment:
```suggestion
func TestOpExtCodeHash_WritesHashInStack(t *testing.T) {
```

The reason is removing judgements and redundancies from names, the result written should always be correct.

Reply:
```suggestion
func TestOpExtCodeHash_QueriesRetrievesAndWritesCodeHashInStack(t *testing.T) {
```

This is if we start checking for the account in the mock arguments, I guess we should do it.

Reply:
I prefer to stay just with `WritesHashInStack`, since we are not really testing, query/retrieve, those are getting mocked.